packages/firebase_crashlytics/android/src/main/java/io/flutter/plugins/firebase/crashlytics/firebasecrashlytics/FirebaseCrashlyticsPlugin.java

```diff
@@ -103,7 +103,8 @@ private StackTraceElement generateStackTraceElement(Map<String, String> errorEle
       String className = errorElement.get("class");
       String methodName = errorElement.get("method");
 
-      return new StackTraceElement(className == null ? "" : className, methodName, fileName, Integer.parseInt(lineNumber));
+      return new StackTraceElement(
+          className == null ? "" : className, methodName, fileName, Integer.parseInt(lineNumber));
```

Comment:
I think `className ?? ""` would be more readable
```suggestion
          className ?? "", methodName, fileName, Integer.parseInt(lineNumber));
```

Reply:
Oops, never mind, this is Java not Dart :)

Reply:
Unless you know something I don't, I don't think that Java has a null coalescing operator yet...

Reply:
But I agree that's definitely more readable 😄